### DISCUSSION TITLE
juicefs转存数据到云端时是否进行校验

### DISCUSSION THREAD

**Discussion by colinblack** [juicedata/juicefs#1880] [Q&A] [answered]
juicdfs将本地磁盘数据转存到云端时是否有校验以及校验失败后的异常处理机制

**Comment by davies** [comment, marked as answer]
是用 sync 来做转存吗？它有一个可选的校验机制，通过 --check-new 或者 --check-all 来开启。

在写入数据到 云上对象存储时，一般 用来写入的 SDK  也会有数据校验的。

**Reply by colinblack** [reply]
好的, 谢谢
